build/Uno.WinUI.nuspec

```diff
@@ -16,6 +16,13 @@
 
 	<dependencies>
 
+	  <!-- net5.0-windows10.0.18362.0 -->
+	  <group targetFramework="$winuitargetpath$">
```

Comment:
This will likely apply to UWP as well. I think it's not intended?

Reply:
I think this should be alright since it is actually referenced by Uno.UI.Toolkit for both:
https://github.com/unoplatform/uno/blob/795aa3a2ad1c9ba4fecd2541119fc57e8a00b92c/src/Uno.UI.Toolkit/Uno.UI.Toolkit.csproj#L84-L94

Reply:
@dansiegel The `PackageReference` condition is `Condition="'$(UNO_UWP_BUILD)'!='true'"`

Reply:
Actually you're right... I missed the condition on the build... @jeromelaban should we just set this to net5.0-windows10.0.18362.0 then?